docs/index.rst

```diff
@@ -15,15 +15,17 @@ speak different languages.
 
 If one has a number of complex systems and wills to combine them to create a larger, even more
 complex system, *SOSS* can act as an intermediate message-passing tool that, by speaking a common
-language, mediates the integration. The inter-systems bridges are centralized and configured with a YAML text file,
-through which the user can provide a mapping between the topics and services on the middlewares of the involved systems.
+language, centralizes and mediates the integration.
```

Comment:
I miss the introduction to the YAML files, and the mapping between the topics and services.

Reply:
They are below!

Reply:
My fault.